src/NServiceBus.Core/Hosting/StartupDiagnostics/Host.cs

```diff
@@ -37,7 +37,7 @@ static string DeriveAppDataPath(Assembly systemWebAssembly)
 
             if (appDataPath == null)
             {
-                throw new Exception(GetMapPathError("Failed since MapPath returned null"));
+                throw new Exception(GetMapPathError("Failed since MapPath returned null."));
```

Comment:
Adding a period here means you've doubled up, since the string on line 69 already has one.

Reply:
I've removed the period on line 69 as the sentence on line 48 was also containing one.